practice-areas/engineering/git.md

```diff
@@ -147,6 +147,151 @@ Here is a recommended Git configuration block that it is recommended to add to y
   ff = only
 [pull]
   ff = only
+[commit]
+  gpgsign = true
+```
+
+## Commit Signing
+
+Commit signing allows for identification and non-repudiation of individual commits to repositories in both Github and Gitlab. It improves security of our repositories by ensuring we know who made changes. The following instructions describe how to create a new GPG keypair, associate it with your Github and Gitlab accounts, and enable commit signing by default with git.
+
+### Creating a new GPG Key
```

Comment:
This section is excellent and IMO should get its own page ("Setting up GnuPG") for GPG in general, and here keep just the lines that are specific to the `.gitconfig` setup and other Git-related commands. I would link to the general GnuPG page from https://guidebook.civicactions.com/en/latest/common-practices-tools/security/encryption/#the-gnu-privacy-guard-gpg

Reply:
Ok, thanks @openprivacy! I split it out into a new gnupg.md page. Let me know if it works.